.github/bin/github-pages-setup.sh

```diff
@@ -97,7 +100,7 @@ Verible tool suite.
 ## Command line arguments
 \`\`\`
 EOF
-bazel-bin/verilog/tools/syntax/verible-verilog-syntax -helpfull >> $SYNTAX_DOC
+${BINDIR}/verible-verilog-syntax -helpfull || true >> $SYNTAX_DOC
```

Comment:
Due to the `|| true`, if the binary does not exist we would actually not notice here and end up with an empty output here...

Maybe we should test the existence of every binary beforehand. Since we have `set -e`, something like the following should be sufficient to exit with an error early in that case:

```
test -x "${BINDIR}/verible-verilog-syntax"
```

Reply:
Done. I created a wrapper function for this.